### PR TITLE
[CUR-654] Notification enhancment 

### PR DESCRIPTION
**Pull request by muhammadqamar**

### CODE REVIEW

File: src/containers/Notification/index.js [hunk 1]
```diff
@@ -24,7 +24,39 @@ const Notification = (props) => {
         </div>
         <div className="content-wrapper">
           <div className="notification-wapper">
-            {notificationData.map((msg) => <NotificationArea content={msg} type="header" />)}
+            {notificationData.today && Object.keys(notificationData.today).length > 0
+              && (
+              <>
+                <div className="notification-status"> Today </div>
+                {notificationData.today.map((msg) => (
+                  <NotificationArea content={msg} type="header" />
```
Comment (simonebelotti75) [inline src/containers/Notification/index.js:32]: Please add key here

File: src/containers/Notification/index.js [hunk 2]
```diff
@@ -24,7 +24,39 @@ const Notification = (props) => {
...
+              )}
+          </div>
+
+          <div className="notification-wapper">
+            {notificationData.yesterday && Object.keys(notificationData.yesterday).length > 0
+              && (
+              <>
+                <div className="notification-status"> Yesterday </div>
+                {notificationData.yesterday.map((msg) => (
+                  <NotificationArea content={msg} type="header" />
```
Comment (simonebelotti75) [inline src/containers/Notification/index.js:44]: Same here

File: src/containers/Notification/index.js [hunk 3]
```diff
@@ -24,7 +24,39 @@ const Notification = (props) => {
...
+              )}
+          </div>
+
+          <div className="notification-wapper">
+            {notificationData.older && Object.keys(notificationData.older).length > 0
+              && (
+              <>
+                <div className="notification-status"> Older </div>
+                {notificationData.older.map((msg) => (
+                  <NotificationArea content={msg} type="header" />
```
Comment (simonebelotti75) [inline src/containers/Notification/index.js:56]: Same here

**Review by simonebelotti75 — CHANGES_REQUESTED** [review]

File: src/containers/Notification/index.js (hunk 1 above)
Comment (muhammadqamar) [reply src/containers/Notification/index.js:32]: added

File: src/containers/Notification/index.js (hunk 2 above)
Comment (muhammadqamar) [reply src/containers/Notification/index.js:44]: added

File: src/containers/Notification/index.js (hunk 3 above)
Comment (muhammadqamar) [reply src/containers/Notification/index.js:56]: added

**Review by simonebelotti75 — APPROVED** [review]

**Pull request merged**